src/main/java/org/folio/service/manager/export/ExportManagerImpl.java

```diff
@@ -44,6 +48,7 @@
   private static final int POOL_SIZE = 2;
   private static final int SRS_LOAD_PARTITION_SIZE = 50;
   private static final int INVENTORY_LOAD_PARTITION_SIZE = 50;
+  private static final String DEFAULT_MAPPING_PROFILE_ID = "25d81cbe-9686-11ea-bb37-0242ac130002";
```

Comment:
No need to store DEFAULT_MAPPING_PROFILE_ID in the ExportManager, just create a method `getDefaultProfile` in the MappingProfileService to retrieve the default profile, store DEFAULT_MAPPING_PROFILE_ID there